cmd/podman/cp.go

```diff
@@ -329,6 +329,9 @@ func copy(src, destPath, dest string, idMappingOpts storage.IDMappingOptions, ch
 
 	destfi, err := os.Stat(destPath)
 	if err != nil {
+		if strings.HasSuffix(dest, string(os.PathSeparator)) {
```

Comment:
looks like the same error messge, can you do a single if with an "or" statement?